bin/install.sh

```diff
@@ -92,25 +92,34 @@ function install_telemetry() {
 # Switch to other istio-control-namespace
 function switch_istio_control() {
     if [ "$ISTIO_CONTROL_NS" != "$ISTIO_CONTROL_OLD" ]; then
-        for ns in $(kubectl get namespaces --no-headers -l istio-env=${ISTIO_CONTROL_OLD} -o=custom-columns=NAME:.metadata.name); do        kubectl label namespaces default --overwrite istio-env=${ISTIO_CONTROL_NS}
+        ACTIVE_NAMESPACES=$(kubectl get namespaces --no-headers -l istio-env=${ISTIO_CONTROL_OLD} -o=custom-columns=NAME:.metadata.name)
+        for ns in $ACTIVE_NAMESPACES; do
             kubectl label namespaces ${ns} --overwrite istio-env=${ISTIO_CONTROL_NS}
         done
-        kubectl set env --all deployment --env="LAST_MANUAL_RESTART=$(date +%s)" --namespace=default
-        kubectl delete namespace $ISTIO_CONTROL_OLD --wait --ignore-not-found
+        if [ $REMOVE_OLD_CONTROL = true ]; then
+            kubectl delete namespace $ISTIO_CONTROL_OLD --wait --ignore-not-found
+        fi
+        for ns in $ACTIVE_NAMESPACES; do
+            kubectl set env --all deployment --env="LAST_MANUAL_RESTART=$(date +%s)" --namespace=$ns
+        done
     fi
 }
 
 COMMAND="install_all"
 METHOD=Install
 ISTIO_CONTROL_OLD=$(kubectl get namespaces -o=jsonpath='{$.items[:1].metadata.labels.istio-env}' -l istio-env)
 ISTIO_CONTROL_OLD=${ISTIO_CONTROL_OLD:-istio-control}
+REMOVE_OLD_CONTROL=false
 
 while [ $# -gt 0 ]
 do
     case "$1" in
         --update)
             METHOD=Update
             ;;
+        --remove-old-control)
```

Comment:
I would prefer we use env variables consistently for options. Easier on the CI/CD and makefile.
The installer will probably move into istioctl anyways - but even there uncommon options should be set as env variables.

One option is for 'installer' to be packed as a docker container, with helm, istioctl, kubect - also easier to customize
various things via -e.